tests/unit_tests/test_lobbyconnection.py

```diff
@@ -247,7 +246,7 @@ async def test_command_game_host_creates_game(
         "game_mode": "faf",
         "game_class": CustomGame,
         "name": test_game_info["title"],
-        "host": players.hosting,
+        "host": players.joining,
```

Comment:
See in this test it makes sense to use the hosting player

Reply:
Everything else looks good. For these tests named `test_command_game_host_*` I would keep it as the hosting player.

Reply:
Meaning this code is correct or it needs to be reversed? It looks to me like both players.hosting and players.joining would work.